frontend/src/components/filter/FiltersButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Button, Theme } from '@material-ui/core';
+import { ExpandMore, ExpandLess } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  filtersButton: {
+    padding: '8px 0',
```

Comment:
This one looks a bit weird. I'd recomment either using the default padding or using 8px all around.

![Screenshot 2021-12-19 at 08 59 59](https://user-images.githubusercontent.com/8818081/146667952-c5e86365-a285-4f61-bf5f-095cc116851e.png)

Reply:
Fixed in d5d3ff2